code/modules/cargo/packs/misc.dm

```diff
@@ -13,17 +13,16 @@
 //////////////////// Paperwork and Writing Supplies //////////////////////////
 //////////////////////////////////////////////////////////////////////////////
 
-/* I did it Kevin
 /datum/supply_pack/misc/abandonedcrate
```

Comment:
I always wanted to add in 10mm guns into cargo, just thought it would never get merged